dhcpv4/client.go

```diff
@@ -124,7 +124,7 @@ func MakeListeningSocket(ifname string) (int, error) {
 // ordered as Discovery, Offer, Request and Acknowledge. In case of errors, an
 // error is returned, and the list of DHCPv4 objects will be shorted than 4,
 // containing all the sent and received DHCPv4 messages.
-func (c *Client) Exchange(ifname string, discover *DHCPv4) ([]DHCPv4, error) {
+func (c *Client) Exchange(ifname string, discover *DHCPv4, modifiers ...Modifier) ([]DHCPv4, error) {
```

Comment:
If I'm understanding this correctly, this will apply `modifiers` to both the DISCOVER and REQUEST packets. Are all potential modifiers applicable to both packets? How would you restrict which modifiers get applied to one or the other? Is that even desired functionality?

Reply:
Yes it will apply to both. 
*Are all potential modifiers applicable to both packets?* I'm not sure.
*How would you restrict which modifiers get applied to one or the other?* With the code in this pull request you can't.
*Is that even desired functionality?* I'm trying to be consistent with [how the DHCPv6 client does it](https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/client.go#L43)

Reply:
Ah, didn't realize this was already in dhcpv6.